snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStore.java

```diff
@@ -271,6 +271,16 @@ public ActorFuture<Set<PersistedSnapshot>> getAvailableSnapshots() {
     return actor.call(() -> Collections.unmodifiableSet(availableSnapshots));
   }
 
+  @Override
+  public ActorFuture<Long> getCompactionBound() {
+    return actor.call(
+        () ->
+            availableSnapshots.stream()
```

Comment:
:thought_balloon: Alternatively, we could keep the available snapshots in a `NavigableMap` so we can easily delete the oldest ones always :shrug:

Reply:
It doesn't matter much as we don't expect to keep more than 2 or 3 snapshots at a time. But if it simplifies the code then go for it.